client/ayon_core/scripts/slates/slate_base/api.py

```diff
@@ -1,15 +0,0 @@
-from .font_factory import FontFactory
```

Comment:
I think removing content of `api.py` in this case is not a good idea. Probably rather add `__all__`.